cub/cub/device/dispatch/tuning/tuning_transform.cuh

```diff
@@ -91,6 +92,15 @@ struct async_copy_policy_t
   static constexpr int bulk_copy_alignment = BulkCopyAlignment;
 };
 
+template <int BlockThreads, int Size1, int... Sizes>
+struct vectorized_policy_t
+{
+  static constexpr int block_threads    = BlockThreads;
+  static constexpr int items_per_thread = 32 / Size1; // TODO(bgruber): we should specify vectors per thread, not items
+  static constexpr int vector_load_length = ::cuda::std::max(8 / Size1, 1); // How many elements in single load
+                                                                            // instruction
```

Comment:
Nitpick: The comment doesnt really add much clarity

Reply:
Dropped during reengineering